tools/build.sh

```diff
@@ -26,9 +26,13 @@ if [ -z "${1:-}" ]; then
     exit 1
 fi
 
+# If no repo is given get it from the ambient project.
 if [ -z "${2:-}" ]; then
-    echo "Must specify the name of the repo."
-    exit 1
+    readonly project_id=$(gcloud config list core/project --format="csv[no-heading](core)" | cut -f 2 -d '=')
```

Comment:
Don't you need to cut with `-s` parameter?
Should you add a check that fails if core/project is not set in the current gcloud config?